drivers/system_pathways/app/models/system_pathways/equity/disabling_condition.rb

```diff
@@ -12,10 +12,10 @@ module SystemPathways::Equity::DisablingCondition
       chart: 'disabling_condition',
       config: {
         size: {
-          height: 800,
+          height: node_names.count * 30,
```

Comment:
Do we need a default height for this (and other similar height calculations) in case it ever equates to zero?

Reply:
Good question.  It default to 100 in the calling file.  I guess if there are no nodes (which can't happen) it might end up not showing up, but the node set is fixed, so I think that's ok.